lib/validators/options_processor/options/must_exclude.js

```diff
@@ -34,9 +35,23 @@ class MustExclude {
         return !regexObj.test(input)
       } else if (Array.isArray(input)) {
         if (filter.all) {
-          return input.every(label => !regexObj.test(label))
+          return input.every(label => {
+            if (typeof label === 'string') {
+              return !regexObj.test(label)
+            } else if (key) {
+              return !regexObj.test(label[key])
+            }
+            return false
```

Comment:
should this return an error instead? If the code reaches here, doesn't this mean that user has probably set some settings wrong?

Reply:
good point, adding now